packages/manager/modules/pci/src/projects/project/quota/quota.html

```diff
@@ -0,0 +1,92 @@
+<pci-project-quota-header></pci-project-quota-header>
+
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div class="text-center" data-ng-if="$ctrl.isLoading">
+     <oui-spinner></oui-spinner>
+</div>
+
+<div class="pt-2" data-ng-if="!$ctrl.isLoading">
+    <!-- THERE IS NO PAYMENT MEAN - SHOW A MESSAGE TO ADD IT -->
+    <div data-ng-if="$ctrl.isQuotaRestricted() && !$ctrl.defaultPaymentMean">
+        <div class="oui-box">
+            <p class="bold" data-translate="pci_projects_project_quota_restricted_account"></p>
+            <p data-translate="pci_projects_project_quota_restricted_account_unlock"
+                data-translate-values="{'url': $ctrl.paymentmeanUrl}">
+            </p>
+        </div>
+    </div>
+
+    <!-- THERE IS A PAYMENT MEAN BUT QUOTA IS RESTRICTED -  -->
+    <div data-ng-if="$ctrl.isQuotaRestricted() && $ctrl.defaultPaymentMean">
+        <div class="oui-box" >
+            <p class="bold" data-translate="pci_projects_project_quota_restricted_paymentmean_active"></p>
+            <button class="oui-button oui-button_primary"
+                    data-ng-click="$ctrl.unleashAccount()">
+                <span data-translate="pci_projects_project_quota_restricted_unlock_button"></span>
```

Comment:
The `translate` directive could be applied directly on the parent `button` element.